runtime/amd64.S

```diff
@@ -448,6 +448,110 @@ CFI_STARTPROC
 CFI_ENDPROC
 ENDFUNCTION(G(caml_allocN))
 
+/* Polling */
+FUNCTION(G(caml_call_poll))
```

Comment:
Rather than copy-pasting this large piece of code, use a macro definition to take in the function to call as an argument and shared the code between `caml_call_gc` and `call_call_poll`.

Reply:
I might have missed something, but why is this function needed at all? Why not just call caml_call_gc here?